main/main.ino

```diff
@@ -123,6 +123,18 @@ unsigned long ReceivedSignal[array_size][2] = {{0, 0}, {0, 0}, {0, 0}, {0, 0}};
 //adding this to bypass the problem of the arduino builder issue 50
 void callback(char* topic, byte* payload, unsigned int length);
 
+#ifdef ESPWifiManualSetup
+char manual_wifi_ssid[] = wifi_ssid;
+char manual_wifi_password[] = wifi_password;
+#endif
```

Comment:
to reduce the memory footprint we can move this into `setup_wifi`

Reply:
Good call. I've moved them to `setup_wifi`